.gitlab-ci.yml

```diff
@@ -2,350 +2,165 @@
-  only:
-    variables:
-      - $DOCKERIMAGE == $CI_JOB_NAME
+  <<:                   *docker_build
+
+query-exporter:
+  <<:                   *docker_build
+
+redis-exporter:
+  <<:                   *docker_build
 
-# build k8s on top of node 12-alphine 
+# build k8s on top of node 12-alphine
 chaostools:
-  <<: *docker_build
+  <<:                   *docker_build
   variables:
-    CONTAINER_IMAGE:               $CI_JOB_NAME
-    CONTAINER_TAG:                 latest
-    DOCKERFILE_DIR:                $CI_JOB_NAME
     # https://github.com/kubernetes/kubernetes/releases
-    BUILD_KUBE_VERSION:            "1.14.6"
-  only:
-    variables:
-      - $DOCKERIMAGE == "chaostools"
+    BUILD_KUBE_VERSION: "1.14.6"
   script:
-    - |
-      cat <<-EOT
-      |
-      | # build of chaostools image
-      |
-      | KUBE_VERSION = ${BUILD_KUBE_VERSION}
-      |
-      EOT
     - cd dockerfiles/
     - docker build
       --build-arg VCS_REF="${CI_COMMIT_SHA}"
       --build-arg BUILD_DATE="$(date -u '+%Y-%m-%dT%H:%M:%SZ')"
+      --build-arg REGISTRY_PATH=$REGISTRY_PATH
       --build-arg KUBE_VERSION="${BUILD_KUBE_VERSION}"
-      --tag parity/$CONTAINER_IMAGE:$CONTAINER_TAG
+      --tag $REGISTRY_PATH/$CONTAINER_IMAGE:$CONTAINER_TAG
       --file $DOCKERFILE_DIR/Dockerfile .
-    - docker push parity/$CONTAINER_IMAGE:$CONTAINER_TAG
+    # Push to Dockerhub
+    - docker login -u "${Docker_Hub_User_Parity}" -p "${Docker_Hub_Pass_Parity}"
+    - docker info
+    - docker push $REGISTRY_PATH/$CONTAINER_IMAGE:$CONTAINER_TAG
+    - docker logout
 
 # special case as version tags are introduced
 kubetools:
-  <<: *docker_build
+  <<:                   *docker_build
   variables:
-    CONTAINER_IMAGE:               $CI_JOB_NAME
-    CONTAINER_TAG:                 latest
-    DOCKERFILE_DIR:                $CI_JOB_NAME
     # https://github.com/kubernetes/kubernetes/releases
-    BUILD_KUBE_VERSION:            "1.14.6"
+    BUILD_KUBE_VERSION:            "1.18.2"
     # https://github.com/kubernetes/helm/releases
     # will be overwritten by the global variable at
     # https://gitlab.parity.io/groups/parity/-/settings/ci_cd
-    BUILD_HELM_VERSION:            "2.14.3"
-  only:
-    variables:
-      - $DOCKERIMAGE == $CI_JOB_NAME
-      - $DOCKERIMAGE == "kubetools"
+    BUILD_HELM_VERSION:            "2.16.6"
   script:
-    - |
-      cat <<-EOT
-      |
-      | # build of kubetools image
-      |
-      | KUBE_VERSION = ${BUILD_KUBE_VERSION}
-      | HELM_VERSION = ${BUILD_HELM_VERSION}
-      |
-      EOT
     - cd dockerfiles/
     - docker build
       --build-arg VCS_REF="${CI_COMMIT_SHA}"
       --build-arg BUILD_DATE="$(date -u '+%Y-%m-%dT%H:%M:%SZ')"
+      --build-arg REGISTRY_PATH=$REGISTRY_PATH
       --build-arg KUBE_VERSION="${BUILD_KUBE_VERSION}"
       --build-arg HELM_VERSION="${BUILD_HELM_VERSION}"
-      --tag parity/$CONTAINER_IMAGE:$CONTAINER_TAG
-      --tag parity/$CONTAINER_IMAGE:$BUILD_HELM_VERSION
```

Comment:
@gabreal is this tag used somewhere? Is it the only reson to make a separate build?